extra/alacritty-msg.man

```diff
@@ -10,20 +10,43 @@ making it possible to control Alacritty without directly accessing it.
 \fB\-s\fR, \fB\-\-socket\fR <socket>
 Path for IPC socket creation
 .SH "MESSAGES"
+.TP
 \fBcreate-window\fR
 Create a new window in the same Alacritty process
 .TP
 .SH "\tOPTIONS"
 .RS 12
+.TP
 \fB\-\-hold\fR
 Remain open after child process exits
-
+.TP
 \fB\-\-working\-directory\fR <working\-directory>
 Start the shell in the specified working directory
-
+.TP
 \fB\-e\fR, \fB\-\-command\fR <command>...
 Command and args to execute (must be last argument)
 .RE
+.TP
+\fBconfig\fR
+Update the Alacritty configuration
+.TP
+.SH "\tARGS"
+.RS 12
+.TP
+\fB<CONFIG_OPTIONS>...\fR
+Configuration file options [example: cursor.style=Beam]
+.RE
+.TP
+.SH "\tOPTIONS"
+.RS 12
+.TP
+\fB\-w\fR, \fB\-\-window\-id\fR <WINDOW_ID>
+Window ID for the new config.
+
+Use `-1` to apply this change to all windows.
```

Comment:
What's the idea behind `-1` and not `-a`, the `-a` could stand for `all`, but one is just one? Kind of strange choice.

Ah, so it seems like you're using it as a negative value, but that would mean that it won't work given that your window id could be `u64` that can't feat into `i64`.

An an option if you really want to keep `-1` you should likely use `i128` for that, so it can hold u64.

Reply:
It's just happened that you have a u32 under the hood on X11, because XID is u32. However on Wayland we just take a pointer, and pointers have pretty high values.

Reply:
> What's the idea behind -1 and not -a, the -a could stand for all, but one is just one? Kind of strange choice.

Much easier to parse and handle. I feel like for the user ultimately it doesn't matter, but if we accept both strings and numbers we have to handle both clap and serde serializers/deserializers for a type that *may* be both. Dealing with potential number parsing crap all over.

> Ah, so it seems like you're using it as a negative value, but that would mean that it won't work given that your window id could be u64 that can't feat into i64.

I'm not sure window ID on unix will ever actually be over i64? That would be quite a massive number already. Going for `i128` just seemed excessive?

> It's just happened that you have a u32 under the hood on X11, because XID is u32. However on Wayland we just take a pointer here, and pointers have pretty high values.

Pretty high values, sure. But > i64 high values? I guess if it's really actual pointers it might be up to u64?